cljfmt/src/cljfmt/core.cljc

```diff
@@ -390,3 +391,19 @@
          (reformat-form (cond-> options
                           alias-map (assoc :alias-map alias-map)))
          (n/string)))))
+
+(def default-linebreak
+  #?(:clj (System/lineSeparator) :cljs \newline))
+
+(defn linebreak-type [text]
```

Comment:
The docstring can be ommitted since this is an internal function, and it should be moved to `cljfmt.diff`. Also call it something like `find-line-separator`.

Reply:
renamed. it's used by both `diff` & `main` so will leave it in core.